doc/source/admin/framework_dependencies.rst

```diff
@@ -190,12 +181,6 @@ Galaxy can create a virtualenv using the adapted virtualenv package. Once a vali
 .. _conda-forge: https://conda-forge.org/
 .. _Bioconda: https://bioconda.github.io/
 
-uWSGI
-^^^^^
-
-``run.sh`` should automatically set ``--virtualenv`` on uWSGI's command line. However, you can override this using the
-``virtualenv`` option in the ``uwsgi`` section of ``galaxy.yml`` as described in the `Managing dependencies manually`_
-section.
 
 Unpinned dependencies
 ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think this section can be removed completely. One could use [poetry](https://python-poetry.org/) directly to do something similar, but I haven't seen anyone request that.